AppManager/app_manager_server.py

```diff
@@ -203,8 +218,56 @@ def start_app(config):
     monit_interface.stop(watch)
     return BAD_PID
 
+  if 'log_size' in config.keys():
+    log_size = config['log_size']
+  else:
+    if config['app_name'] == APPSCALE_DASHBOARD_ID:
+      log_size = TEN_MB
```

Comment:
Instead of having a constant for the dashboard, can we pass in log_size for the dashboard from the caller?